src/formMapper.spec.js

```diff
@@ -193,6 +193,23 @@ describe('map values', () => {
         expect(retVal).toEqual(comparisonVal);
     });
 
+    it('should handle file inputs', () => {
+        const comparisonVal = {
+            input1: ['1.jpg', '2.jpg']
+        };
+        const data = [
+            new File(["1"], "1.jpg"), new File(["2"], "2.jpg")
+        ];
+        document.body.innerHTML = `
+            <form>
+                <input type="file" multiple="multiple" />
+            </form>
+        `;
+        const formEl = document.querySelector('form');
+        const retVal = formMapper.getValues(formEl);
+        expect(retVal).toEqual(comparisonVal);
```

Comment:
I am not 100% sure how to test a file list, but some alternative expect statements could be

```
expect(retVal).toHaveProperty({{inputName}})
expect(retVal[name]).toBeInstanceOf(FileList)
```
For reference:
https://jestjs.io/docs/en/expect.html#tohavepropertykeypath-value
https://jestjs.io/docs/en/expect.html#tobeinstanceofclass